src/main/java/com/wuest/prefab/gui/GuiBase.java

```diff
@@ -121,6 +121,23 @@ public ExtendedButton createAndAddButton(int x, int y, int width, int height, St
         return this.addDrawableChild(returnValue);
     }
 
+    /**
+     * Creates a button using the button clicked event as the handler. Then adds it to the buttons list and returns the created object.
+     *
+     * @param x      The x-axis position.
+     * @param y      The y-axis position.
+     * @param width  The width of the button.
+     * @param height The height of the button.
+     * @param text   The text of the button.
+     * @param tooltip Should render the tooltip
+     * @return A new button.
+     */
+    public ExtendedButton createAndAddButton(int x, int y, int width, int height, boolean tooltip, String text) {
```

Comment:
The "tooltip" variable has incorrect casing. It should be "toolTip"

In addition; it should indicate by its name that it's a true/false value.

In this case; it should be called "showToolTip".